### DISCUSSION TITLE
Adaptive Card render not showing

### DISCUSSION THREAD

**Discussion by lakshmikanth27** [microsoft/AdaptiveCards#8677] [Q&A]
I changed the botframework version to 4.15.1 to render the tables(adaptive card schema 1.5), it is not showing the table but just a empty card.

tried these:

1. Added ```version``` property to adaptiveCardHostConfig no use, 
2. According to docs under the styleOptions wanted me to add adaptiveCardsParserMaxVersion to mention the version, it did not work either.

Need some inputs to get this solved

**Comment by jwoo-msft** [comment]
Hi @lakshmikanth27,
We have stopped developing for .Net version of AdpativeCards. That is why Table is not being rendered.

**Reply by lakshmikanth27** [reply]
Hi @jwoo-msft I was asking for javascript not .Net sorry I did not mention about it.

**Comment by jwoo-msft** [comment]
> Hi @jwoo-msft I was asking for javascript not .Net sorry I did not mention about it.

as you can verify in [our designer](https://adaptivecards.io/designer/), tables render correctly. this is not JS AdaptiveCards issue.
